base/strings/io.jl

```diff
@@ -96,6 +96,9 @@ tostr_sizehint(x::Float64) = 20
 tostr_sizehint(x::Float32) = 12
 
 function print_to_string(xs...; env=nothing)
+    if isempty(xs)
+        return ""
+    end
```

Comment:
`isempty(xs) && return ""` would be more idiomatic (and shorter).

Reply:
I think it is a personal taste rather than an idiom. I'd like to use `if` in this kind of case because it is easier to read control flow.

Reply:
The `&&` syntax is widely used in Base to check for small corner cases at the beginning of functions. Hence I would say it is idiomatic in Base. (The word “idiomatic” refers to choices that are questions of taste/usage rather than grammar.)

Obviously the two are technically equivalent, so I won’t insist.

Reply:
(The question in cases like this is not your *personal* taste, but rather the dominant style of the project being patched.)

Reply:
But I can find lots of `if <condition> <newline> <single statement> <newline> end` usage in Base (I don't know stats on which is more dominant) and cannot find a guideline on the style, so I used the words "personal taste" here. Anyway, I think this is off-topic.

Reply:
Both styles are definitely acceptable.